### PR TITLE
Package local cache not found on Linux

### PR DESCRIPTION
**Pull request by alesebi91**
Fixed package search in local filesystem on Linux by checking OS type.

Resolved issue #70

### CODE REVIEW

File: src/Methods.cs [hunk 1]
```diff
@@ -91,61 +96,66 @@ public async Task<PackageList> GetNugetInformationAsync(string project, IEnumera
                             continue;
                         }
 
-                        // Search nuspec in local cache
+                        // Search nuspec in local cache (Fix for linux distro)
                         string userDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
-                        var nuspecPath = Path.Combine(userDir, ".nuget", "packages", packageWithVersion.Name, version, packageWithVersion.Name + ".nuspec");
+
+                        //Linux: package file name is lowercase
+                        var packageNameTmp = IsLinux() ? packageWithVersion.Name?.ToLowerInvariant() : packageWithVersion.Name;
```
Comment (tomchavakis) [inline src/Methods.cs:103]: I am not sure if there is any difference between linux and windows for nuget. I've checked both Ubuntu and Windows packages folder .nuget\packages and both package names are the same. Is there any example ?

**Comment by tomchavakis** [comment]
Hi @alesebi91,

Thank you for the contribution. 
Is it possible to add a Unit Test for the IgnoreSslCertificateErrors please?
Moreover, I am curious why you need to disable the SSL Verification and the only reason that I can think of is that you are using a local nuget server without SSL in your stack. IS that correct ?

**Comment by alesebi91** [comment]
Implemented double check on linux OS to test lowercase path if standard fail.

I'm using this tool inside Microsoft SDK Ubuntu and Debian images where package cache path is lowercase:

> root@xxxxx:~/.nuget/packages# ls
microsoft.csharp
microsoft.data.sqlclient
microsoft.data.sqlclient.sni.runtime
microsoft.identity.client
microsoft.identitymodel.jsonwebtokens
...

Tks!

**Comment by tomchavakis** [comment]
@alesebi91 , 
Please resolve the conflicts to merge it

**Review by tomchavakis — APPROVED** [review]

**Pull request merged**